identity/README.md

```diff
@@ -300,6 +300,27 @@ message Capability {
     Type type = 1;
   }
 
+  // EncryptionKeyRotation contains the features of the EncryptionKeyRotation
+  // operation that the CSI-driver supports.
+  message EncryptionKeyRotation {
+    // Type describes a CSI Service that CSI-drivers can support.
+    enum Type {
+      // UNKNOWN indicates that the CSI-driver does not support the
+      // EncryptionKeyRotation operation in the current mode.
+      // The CSI-Addons CO plugin will most likely ignore this node
+      // for the EncryptionKeyRotation operation.
+      UNKNOWN = 0;
+      // ENCRYPTIONKEYROTATION indicates that the CSI-driver provides
+      // RPCs for an EncryptionKeyRotation operation.
```

Comment:
`CSI-driver` to `CSI driver` please correct this in other places as well.

Reply:
Ack.